tools/shared.py

```diff
@@ -1153,6 +1153,8 @@ def load(self, args=[]):
 
       if get_llvm_target() == WASM_TARGET:
         self.attrs['WASM_BACKEND'] = 1
+      if 'EMCC_EXPERIMENTAL_USE_LLD' in os.environ:
+        self.attrs['EXPERIMENTAL_USE_LLD'] = os.environ['EMCC_EXPERIMENTAL_USE_LLD']
```

Comment:
Why is this an environment variable?
Before we needed to update the LLVM_TARGET, which wasn't doable from Settings (without major grossness), but here it looks like it's more work to plumb through via the env. So how about letting this just be a setting?